src/app/partners/partner/partner.component.ts

```diff
@@ -33,5 +56,39 @@ export class PartnerComponent implements OnInit {
       this.programs = programViews;
       this.programs.sort(UtilsService.comparisonFunction2);
     });
+
+
+    this.contactForm = new FormGroup({
+      name: new FormControl('', Validators.required),
+      number: new FormControl(),
+      email: new FormControl(),
+      title: new FormControl(),
+      organization: new FormControl()
+    });
+
+
+
+  }
+
+  public createContact() {
+    if (!this.contactForm.valid){
+      console.log('Form is not valid');
+    }
+    else{
+      console.log(this.contactForm.value)
+    }
+    // this.createEntity('contact', this.contactForm).then((entityResult: Object) => {
```

Comment:
Try your best to keep commented out code out of the commits. I will be sure to take it out before this is merged though.